src/lit-html.ts

```diff
@@ -435,7 +435,7 @@ const isDirective = (o: any) =>
  * A sentinel value that signals that a value was handled by a directive and
  * should not be written to the DOM.
  */
-export const directiveValue = {};
+export const noChange = {};
```

Comment:
Add:

```ts
/**
 * @deprecated Use `noChange` instead.
 */
export {noChange as directiveValue};
```

so this isn't a breaking change

Reply:
yeah makes sense, done